proto/storage/cve.proto

```diff
@@ -144,6 +145,7 @@ message ImageCVE {
   float nvdcvss = 10; // @gotags: search:"NVD CVSS",store"
   // cvss_metrics stores list of cvss metrics from different sources like Redhat, NVD etc
   repeated CVSSScore cvss_metrics = 11;
+  ScoreVersion nvd_score_version = 12;
```

Comment:
maybe add a comment here says NVD score is not available when nvd_score_version is not set